### PR TITLE
Aman

### PR DESCRIPTION
**Pull request by amandemo1411**
I have added a new class responsible for handling crud operations

### CODE REVIEW

File: LearningDemo/CollectionCreation.cs [hunk 1]
```diff
@@ -6,5 +6,6 @@ namespace LearningDemo
 {
     class CollectionCreation
     {
+        //have to create collections here
```
Comment (bikrajlama) [inline LearningDemo/CollectionCreation.cs:9]: Need to implement

File: LearningDemo/CrudOperation.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningDemo
+{
+    public class CrudOperation
+    {
+
```
Comment (bikrajlama) [inline LearningDemo/CrudOperation.cs]: Please create some methods

File: LearningDemo/CollectionCreation.cs (hunk 1 above)
Comment (amandemo1411) [reply LearningDemo/CollectionCreation.cs:9]: Implemented

File: LearningDemo/CrudOperation.cs (hunk 2 above)
Comment (amandemo1411) [reply LearningDemo/CrudOperation.cs]: Implemented the suggested changes

**Review by bikrajlama — APPROVED** [review]

**Pull request merged**